emperor/support_files/js/controller.js

```diff
@@ -266,6 +266,25 @@ define([
     });
     this.$plotSpace.append(this.$optionsButton);
 
+    /**
+     * @type {Node}
+     * jQuery object
+     */
+    this.$figureModeButton = $('<button name="dm-button">&nbsp;</button>');
+    this.$figureModeButton.css({
+      'position': 'absolute',
+      'z-index': '3',
+      'top': '5px',
+      'right': '40px'
+    }).attr('title', 'Figure Mode').on('click', function(event) {
+        scope.controllers.axes.updateVisibleAxes(null, 2);
```

Comment:
For updating different attributes of the axes controller, I would suggest using `scope.controllers.axes.fromJSON`, then you encode these attributes as an object and you should be good to go (see the unit tests for some examples).